UCI-IN4MATX-191-Token-ATM/token-atm-spa
# Enhancement text formatting

Proofread and updated formatting for strings/text added since #48.

Searched through the `src/` folder for strings that matched the regex expression `` '.*'|".*"|`.*` ``.

No Token ATM Functionality should change.

**Testing Note**
1. The `Current Token Balance: ` display in `student-record-display-component` (`ts` & `html`)
 is the only programming change. [And this code change will be removed once the pluralize utility is accessible through an angular pipe in future]
2. Errors with the `Action Needed` heading now uses a programmatic string template. Errors that use the `actionNeededTemplate(` and `tokenATMContentList(` should have their error modal visually confirmed as readable.
3. (Optional) A newline has been added to the modal message for the errors that require a manual `"Save It Now"` action be done on Canvas. The added spacing should be checked to see if it is an improvement to the UI.
-----
Work Left for future pull requests:
- Create an Angular Pipe for the pluralize utilities
- Thoroughly Review all .html files
- Update name of the Encryption Page (requires versioning/compatibility handling so old versions of Token ATM can still be used)
- Update names in Token Option Creation/Editing (mostly around stating if it's related to Token ATM, Canvas, or another external system)
